moto/ec2/responses/instances.py

```diff
@@ -450,6 +450,7 @@ def _security_grp_instance_attribute_handler(self):
                     </blockDeviceMapping>
                     <virtualizationType>{{ instance.virtualization_type }}</virtualizationType>
                     <clientToken>ABCDE1234567890123</clientToken>
+                    {% if instance.get_tags() != [] %}
```

Comment:
Please make this `if not instance.get_tags()` instead.

Reply:
I have updated the repository to fix this.

Reply:
Sorry! I'm a moron! It should be `if instance.get_tags()` and then list the tags.

Reply:
fixed it again. in all fairness it slipped passed me aswell